### PR TITLE
2016 02 16 np create topics

### PR DESCRIPTION
**Pull request by np**
Please ignore the merge commits, I'll rebase of master when these will be merged.

### CODE REVIEW

**Comment by Mikolaj** [comment]
Is it ready for review? I haven't looked at the generated page, but the code LGTM.

File: src/Frontend/Core.hs [hunk 1]
```diff
@@ -208,11 +208,6 @@ redirectFormHandler
 redirectFormHandler action page processor = formRedirectH action p1 p2 r
   where
     p1 = makeForm page
-
-    p2 result = do
-        void $ processor result
-        return $ redirectOf page
-
+    p2 result = processor result $> redirectOf page
```
Comment (andorp) [inline src/Frontend/Core.hs]: Thx :)

**Comment by andorp** [comment]
I have checked the page, it seems fine. If you consider it done,LGTM.

**Comment by fisx** [comment]
@np: merge?  (should we have merged without getting back with you?  not sure..)

**Pull request merged**